osgi-test/src/test/java/org/ehcache/osgi/OsgiTestUtils.java

```diff
@@ -44,17 +46,37 @@ public static Option baseConfiguration(String ... path) {
       gradleBundle("org.slf4j:slf4j-simple").noStart(),
       gradleBundle("org.apache.felix:org.apache.felix.scr"),
       systemProperty("pax.exam.osgi.unresolved.fail").value("true"),
-      systemPackages(
-        "javax.xml.bind;version=2.3.0",
-        "javax.xml.bind.annotation;version=2.3.0",
-        "javax.xml.bind.annotation.adapters;version=2.3.0"
-      ),
       cleanCaches(true),
       workingDirectory(join(File.separator, "build", "osgi-container", join(File.separator, path))),
       junitBundles()
     );
   }
 
+  public static Option jaxbConfiguration() {
+    if (parseInt(getProperty("java.version").split("[^\\d]+")[0]) >= 9) {
```

Comment:
Any reason you didn't use `\D` instead of `^\d`?

Reply:
Because I find the whole concept of `\D` being the inverse of `\d` abhorrent.